config.json

```diff
@@ -800,6 +800,16 @@
         "strings"
       ]
     },
+    {
+      "slug": "resistor-color-trio",
+      "uuid": "7170d6a3-a32f-44d7-b82e-524485c58aaf",
+      "core": false,
+      "difficulty": 5,
+      "topics": [
+        "conditionals",
+        "loops"
+      ]
+    },
```

Comment:
Unlocked_by? :-)

Reply:
huh. That should be there. It magically disappeared. But I think `"bob"`

Reply:
Done!